include/mrdox/Plugin.hpp

```diff
@@ -12,16 +12,36 @@
 #define MRDOX_PLUGIN_HPP
 
 #include <mrdox/Platform.hpp>
-#include <mrdox/Generator.hpp>
+#include <mrdox/Support/Error.hpp>
+
+#include <string_view>
 
 namespace clang {
 namespace mrdox {
 
+class Generators;
+class Config;
+
+struct Plugin
```

Comment:
But what is this type? it should be `class` according to clang style. But why is it here? Is this really just a helper for a plugin? If so, then it should be in the sources of the example plugin and not part of mrdox public API

Reply:
Sorry, old habit.